notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Fix to "as the docker command"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>